metadata-ingestion/src/datahub/ingestion/source/sql_queries.py

```diff
@@ -71,6 +104,18 @@ class SqlQueriesSourceConfig(
         default=True,
         hidden_from_docs=True,
     )
+    lazy_schema_resolver: bool = Field(
+        description="Use lazy schema loading instead of bulk initialization. When enabled, schemas are fetched on-demand and cached for future lookups. This significantly improves performance for large platforms.",
+        default=True,
+    )
```

Comment:
If the lazy approach is always better, is there any reason to keep the bulk approach?